test/end-to-end/config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "root": "http://localhost:3000/",
+    "api": {
+        "uri": "http://localhost:4000/",
```

Comment:
Is it possible to read the root URL (hostname and port) from an ENV variable and default to localhost? I'm thinking ahead of time when we want this to be running against a docker image.